unreleased/hostmetricsrec-rm-direction.yaml

```diff
@@ -0,0 +1,28 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: hostmetricsreceiver
+
+# A brief description of the change
+note: "Remove direction attribute. The feature gate: `receiver.hostmetricsreceiver.removeDirectionAttribute` can be set to apply the following"
+
+# One or more tracking issues related to the change
+issues: [11820]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be inserted directly into the document, so must be formatted accordingly.
+# Use "subtext: |2" to preserve two spaces of indentation relative to the current level
```

Comment:
Should we maybe just add the indentation in the code instead?

Reply:
Yeah, that's better.